### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,24 +1,7 @@
...
-    read_from_slice,
-};
-pub use types::{
-    AudioTag,
-    Frame,
-    MP3Metadata,
-    OptionalAudioTags,
-    Url,
+pub use crate::{
+    enums::{ChannelType, Copyright, Emphasis, Error, Genre, Layer, Status, Version, CRC},
```
Comment (GuillaumeGomez) [inline src/lib.rs:2]: I strongly dislike this way of writing reexports... Also, what's the point of putting everything inside `pub use crate::`?

File: src/metadata.rs [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-use std::io::Read;
-use std::path::Path;
-use std::time::Duration;
-
-use consts::{BITRATES, SAMPLING_FREQ};
-use enums::{ChannelType, Copyright, CRC, Emphasis, Error, Genre, Layer, Status, Version};
-use types::{AudioTag, Frame, MP3Metadata, OptionalAudioTags};
-use utils::{compute_duration, create_utf8_str, get_line, get_samp_line, get_text_field, get_text_fields};
-use utils::{get_url_field, get_url_fields};
+use crate::{
```
Comment (GuillaumeGomez) [inline src/metadata.rs:1]: Same: the imports are now harder to read. Having `use crate::{` makes no sense.

File: src/utils.rs [hunk 3]
```diff
@@ -1,9 +1,10 @@
+use crate::{
```
Comment (GuillaumeGomez) [inline src/utils.rs:1]: Same.

File: tests/common.rs [hunk 4]
```diff
@@ -1,24 +1,23 @@
-extern crate reqwest;
-
-use std::fs::File;
-use std::io::prelude::*;
-use std::path::Path;
+use std::{fs::File, io::prelude::*, path::Path};
```
Comment (GuillaumeGomez) [inline tests/common.rs:1]: Same issue. :)

**Review by GuillaumeGomez — COMMENTED** [review]
Apart from the import/reexport style, it's a nice improvement overall, thanks! Please just fix them and then it's good to go.

File: src/lib.rs (hunk 1 above)
Comment (flosse) [reply src/lib.rs:2]: this was done by "cargo fix --edition" so it's the convention of the Rust language

Comment (GuillaumeGomez) [reply src/lib.rs:2]: It's definitely not. It's the first time I ever saw it.

Comment (flosse) [reply src/lib.rs:2]: ok, i just tried it again:

if you run `cargo fix --edition` with cargo 1.47.0 on this:

```rust
mod bar;
mod baz;

pub use bar::*;
pub use baz::*;
```

It transforms to
```rust
mod bar;
mod baz;
 
pub use crate::bar::*;
pub use crate::baz::*;
```

Comment (GuillaumeGomez) [reply src/lib.rs:2]: That's pretty bad...

**Pull request closed without merging**